examples/lusid/returns/1-create-transaction-portfolio.sql

```diff
@@ -0,0 +1,30 @@
+-- ============================================================
+-- Description:
+-- 1. In this query, we create a Transaction Portfolio in LUSID
+-- ============================================================
+
+-- Defining scope and code variables
+@@portfolioScope =
+
+select 'IBOR';
+
+@@portfolioCode1 =
```

Comment:
Can you rename this to portfolioCode and drop the 1 as there is one 1 here.

Reply:
Renamed.